subprojects/native/src/test/groovy/org/gradle/internal/nativeintegration/filesystem/services/AbstractFileMetadataAccessorTest.groovy

```diff
@@ -134,6 +149,7 @@ abstract class AbstractFileMetadataAccessorTest extends Specification {
         stat.type == FileType.Missing
         stat.lastModified == 0
         stat.length == 0
+        sameAccessType(stat, VIA_SYMLINK)
```

Comment:
Let's call this `assertSameAccessType()` to have a verb in there.

Reply:
Changed to `assertSameAccessType` and moved the assertion inside the method.